odoo/addons/base/views/res_users_views.xml

```diff
@@ -167,6 +167,15 @@
                                 <field name="active" widget="boolean_button" options='{"terminology": "active"}'/>
                             </button>
                         </div>
+                        <div class="alert alert-info text-center o_form_header" attrs="{'invisible': [('active', '=', True)]}">
+                          <a class="close" data-dismiss="alert" href="#" aria-hidden="true">x</a>
+                          <div>
+                            <strong>The partner linked to this user will remain active</strong>
```

Comment:
Given that this message will be persistent when viewing an archived user, and not just displayed as the result of an action, it would make sense to use the present tense. Also, same terminology comment.
E.g.:   "_The contact linked to this user is still active. You can archive it [here]._"